src/v/debug_bundle/metadata.h

```diff
@@ -0,0 +1,151 @@
+/*
+ * Copyright 2024 Redpanda Data, Inc.
+ *
+ * Use of this software is governed by the Business Source License
+ * included in the file licenses/BSL.md
+ *
+ * As of the Change Date specified in that file, in accordance with
+ * the Business Source License, use of this software will be governed
+ * by the Apache License, Version 2.0
+ */
+
+#pragma once
+
+#include "bytes/bytes.h"
+#include "container/fragmented_vector.h"
+#include "debug_bundle/types.h"
+#include "serde/envelope.h"
+#include "serde/rw/envelope.h"
+#include "serde/rw/named_type.h"
+#include "serde/rw/variant.h"
+#include "utils/uuid.h"
+
+#include <seastar/core/sstring.hh>
+#include <seastar/util/process.hh>
+#include <seastar/util/variant_utils.hh>
+
+#include <chrono>
+#include <variant>
+
+void tag_invoke(
+  serde::tag_t<serde::read_tag>,
+  iobuf_parser& in,
+  uuid_t& uuid,
+  const std::size_t bytes_left_limit);
+
+void tag_invoke(serde::tag_t<serde::write_tag>, iobuf& out, uuid_t uuid);
+
+namespace debug_bundle {
+
+struct process_wait_exited
+  : serde::envelope<
+      seastar::experimental::process,
```

Comment:
this should be process_wait_exited

Reply:
probably an autocomplete fat finger mistake, thanks